packages/office-ui-fabric-react/src/components/Facepile/Facepile.scss

```diff
@@ -1,22 +1,32 @@
 @import '../../common/common';
 
+$spacingAroundItemButton: 4px;
+
 .root {
-  position: relative;
   width: auto;
 }
 
 .clear {
   clear: both;
 }
 
+.members {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  overflow: hidden;
+  margin: -1 * $spacingAroundItemButton;
+
+  > * {
```

Comment:
Hmm I guess its not clear due to the layout of your css.

.itemButton is a child of .members which is given a margin of $spacingAroundItemButton. The parent gives a negative margin of the same amount which results in the parent looking as if it has 0px margin.

I can move the addition of the margin from .itemButton into this child selector which should make this combination more understandable.

When you say it may cause layout breaks in other apps... Since the child is adding the margin back to the parent, there should be no visible layout issues. It's a bit trippy to debug through the visual editor in IE or Chrome but I've never noticed issues in my history of using this trick.

Here's the stackoverflow I created for this exact hack actually:
http://stackoverflow.com/questions/39499067/css-fixed-size-margin-only-between-elements